### DISCUSSION TITLE
Automatically `direnv allow`

### DISCUSSION THREAD

**Discussion by guettli** [direnv/direnv#1115] [General]
Is there a way to configure direnv, so that you don't need an explicit `direnv allow`.

It would be great if this could be set per directory.

I think this would be a compromise: If you enter a new directory (and the .envrc file in it was never executed), then asking for `direnv allow` should happen. But later, if the file got updated, then I don't want to allow the changes again.

Don't get me wrong: I don't want to change the default. The default can be like it is now, but I would like to have a solution for cases where the .envrc file changes a lot.

**Comment by jordanbreen28** [comment]
+1 for this.

**Comment by guettli** [comment]
I found this solution:

```
[whitelist]
prefix = [ "/home/guettli/foo", "/home/guettli/bar" ]
```

Docs: https://direnv.net/man/direnv.toml.1.html#whitelist
